services-api/src/main/java/io/scalecube/services/registry/api/RegistryEvent.java

```diff
@@ -0,0 +1,44 @@
+package io.scalecube.services.registry.api;
+
+import io.scalecube.services.ServiceReference;
+import io.scalecube.transport.Address;
+
+public class RegistryEvent {
+
+  public enum Type {
+    ADDED, REMOVED;
+  }
+
+  private ServiceReference serviceReference;
+  private Type type;
+
+  public RegistryEvent(Type type, ServiceReference serviceReference) {
```

Comment:
Make it private plz.

Reply:
Make a shortcut factory methods: `createAdded()` and `createRemoved()` , sort of. Not neccessarly with that names.